src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+  * Function takes a string with styles
+  * (see an example in [stylesString.js](./stylesString.js))
+  * and returns an object where CSS properties are keys
+  * and values are the values of related CSS properties (see an exampl
+  * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssRulesObject = {};
+  const cssRulesArray = sourceString.split(';')
+    .map(cssItem => cssItem.slice(1).split(':'))
```

Comment:
put first split on the next line 
![image](https://user-images.githubusercontent.com/75431691/226908147-02a1a00a-8758-4102-95bd-b7948acd7587.png)

Reply:
agree